src/Xamarin.Android.Build.Tasks/Xamarin.Android.CSharp.targets

```diff
@@ -37,6 +37,8 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
         <IsXBuild Condition="'$(MSBuildRuntimeVersion)' == ''">true</IsXBuild>
         <EnableDefaultOutputPaths Condition=" '$(EnableDefaultOutputPaths)' == '' And '$(OS)' != 'Windows_NT' ">false</EnableDefaultOutputPaths>
         <EnableDefaultOutputPaths Condition=" '$(EnableDefaultOutputPaths)' == '' ">true</EnableDefaultOutputPaths>
+        <!-- Enable nuget package conflict resolution -->
+        <ResolveAssemblyConflicts>true</ResolveAssemblyConflicts>
```

Comment:
XA.CS+FS, they import:

```
 48     <Import Project="Xamarin.Android.DefaultOutputPaths.targets" Condition="'$(IsXBuild)' != 'true' and '$(EnableDefaultOutputPaths)' == 'true'" />
 49     <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
 50     <Import Project="Xamarin.Android.Common.targets" />
```

`Xamarin.Android.Common.props` gets imported at line 50, which is after `Microsoft.CSharp.targets` and too late.
I added it to `Xamarin.Android.Common.props` so that it gets set when importing `Xamarin.Android.Bindings.targets`.